npm/ng-packs/packages/theme-basic/src/lib/components/page-alert-container/page-alert-container.component.html

```diff
@@ -4,10 +4,12 @@
     [ngClass]="{ 'alert-dismissible fade show': alert.dismissible }"
     role="alert"
   >
-    <h4 class="alert-heading" *ngIf="alert.title">
-      {{ alert.title | abpLocalization: alert.titleLocalizationParams }}
-    </h4>
-    {{ alert.message | abpLocalization: alert.messageLocalizationParams }}
+    <h4
+      class="alert-heading"
+      *ngIf="alert.title"
+      [innerHTML]="alert.title | abpLocalization: alert.titleLocalizationParams"
```

Comment:
Can you search/check innerHtml might be XSS issue. We should sanitize the content of html.

Reply:
Please if apply any changes, reflect changes lepton-x side too.